exe/node/Main.hs

```diff
@@ -112,7 +113,8 @@ main = do
     keys        <- runReaderT readKeyPair keysPath
     nid         <- pure (mkNodeId $ fst keys)
     mem         <- Mempool.newIO
-    gen         <- Yaml.decodeFileThrow genesis :: IO (Block RadTx Nakamoto.PoW)
+    gen         <-
+        Yaml.decodeFileThrow genesis :: IO (Block RealWorldCrypto (RadTx RealWorldCrypto) (Sealed RealWorldCrypto Nakamoto.PoW))
```

Comment:
Can we maybe break this line up differently? It's verrry long. Alternatively, maybe type applications would be shorter?

Reply:
I am wondering if we could simply type alias this long type (in an ad-hoc way in this module) and use that 🤔

Reply:
9a927aa looks like `GenesisBlock` is fitting here.